include/ginkgo/core/base/index_set.hpp

```diff
@@ -169,7 +190,7 @@ class IndexSet : public EnablePolymorphicObject<IndexSet<IndexType>> {
      * @param local_index  the local index.
      * @return  the global index from the index set.
      *
-     * @warning This single entry query can have significant kernel lauch
```

Comment:
![orihime-inoue-bleach](https://user-images.githubusercontent.com/1693511/158887098-161662aa-c4bb-4238-887d-7f2bc339e824.gif)

Reply:
I had completely forgotten about this. All the weird dubs that people made over this was hilarious.